packages/core/src/tracing/instrumentation/cloud/aws-sdk/v3/index.js

```diff
@@ -81,14 +82,25 @@ function shimSmithySend(originalSend) {
     const command = smithySendArgs[0];
 
     if (isActive) {
-      const awsProduct = operationMap[command.constructor.name];
-
-      if (awsProduct) {
-        return awsProduct.instrumentedSmithySend(self, originalSend, smithySendArgs);
+      if (self.config.serviceId) {
```

Comment:
nitpik: Wondering if `self.config` could be undefined for any older v3 version

Reply:
Done